internal/servegit/serve.go

```diff
@@ -248,4 +251,4 @@ func explainAddr(addr string) string {
 See https://docs.sourcegraph.com/admin/external_service/src_serve_git for
 instructions to configure in Sourcegraph.
 `, addr)
-}
+}
```

Comment:
your editor is hungry and its favourite snack is EOL at EOF

Reply:
To be fair to my hungry hungry editor, Im not sure why the lintint tool likes newlines at the end of files so much 😅 .

Reply:
I didn't know this until now, but apparently it's a POSIX standard - that a line not ending with a newline character is not a "line" per se. This stackoverflow Q&A has a lot of context: https://stackoverflow.com/a/729795/1773961.

Reply:
The only reason I personally prefer newlines at the end of a file is so that if I'm ever looking at them in a terminal, my terminal prompt does not get wrapped into the ending of the file and I can have it in a completely new line. (Although in recent years I've added custom script to ensure a new line is always added by ZSH even if files don't end in a newline to handle the edge case, but that's just me. 😁)